tests/snapshots/TestCLICommands_Valid_Log_Level_in_Config_File.stderr.golden

```diff
@@ -1,3 +1,3 @@
-DEBU Set logs-level=debug logs-file=/dev/stderr
+TRCE Set logs-level=trace logs-file=/dev/stderr
```

Comment:
This is wierd to me. We shouldn't dynamically set the log level of the `log` output based on the configuration.

Reply:
Ok, never mind. I thought we were doing something dynamically, such that if I set Log level to `Info`, that this would then say:
```
INFO Set logs-level=info logs-file=/dev/stderr
```

Please confirm this is _not_ the case.